quadcopter_brain/src/quadcopter_brain/landing_site.py

```diff
@@ -70,6 +71,29 @@ def lat_lon(self, copter):
                                             absolute_site[0][0],
                                             absolute_site[1][0])
 
+    def get_average_lat_long(self, copter, total_time=5.0, time_step=0.1):
+        '''
+        Waits for 'time' seconds and samples landing site position
+        every 'time_step' seconds. Returns the average gps position
+        over that time period. Returns None if fiducial never seen
+        '''
+        landing_site_lat = []
+        landing_site_long = []
+        time_limit = datetime.timedelta(seconds=total_time)
+        time_end = datetime.datetime.now() + time_limit
+        while datetime.datetime.now() < time_end:
+            if self.in_view:
+                current_lat, current_long = self.lat_long(copter)
+                landing_site_lat.append(current_lat)
+                landing_site_long.append(current_long)
+            else:
+                print("Averaging landing site GPS, couldn't see fiducial")
+            rospy.sleep(time_step)
+        if len(landing_site_lat) > 0:
+            return (np.mean(landing_site_lat), np.mean(landing_site_long))
+        else:
+            return None, None
```

Comment:
does this absolutely need to be a tuple for something (and None will not work)? If not, just get rid of the else.